src/components/Accordion/Accordion.js

```diff
@@ -10,19 +11,26 @@ export const Container = styled.div`
 `;
 
 const Label = styled.div`
-  padding: ${({ theme }) => theme.accordion.padding};
-  color: ${({ theme }) => theme.accordion.labelFG};
-  font-family: ${({ theme }) => theme.accordion.labelFontFamily};
-  font-size: ${({ theme }) => theme.accordion.labelFontSize};
-  text-transform: ${({ theme }) => theme.accordion.textTransform};
-  font-weight: ${({ theme }) => theme.accordion.labelFontWeight};
+  padding: ${({ theme, mods }) => mods.small ? theme.spacing.medium : theme.spacing.large};
+  color: ${({ theme, mods }) => mods.small ? theme.colors.black : theme.colors.primary};
+  font-family: ${({ theme }) => theme.fonts.brand};
+  font-size: ${({ mods }) => mods.small ? '14px' : '1.5em'};
```

Comment:
It seems like it would be helpful to pull the font sizes from a set of font sizes in the theme. This would ensure that fonts are consistent with our grid system.